extensions/wikia/Discussions/DiscussionsHooksHelper.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+class DiscussionsHooksHelper {
+	/**
+	 * IRIS-5184: Exclude outgoing links in Forum content from Special:WhatLinksHere
+	 * @see SpecialWhatLinksHere::showIndirectLinks()
+	 *
+	 * @param array $pageLinkCondition
+	 */
+	public static function onSpecialWhatLinksHereBeforeQuery( array &$pageLinkCondition ) {
+		$pageLinkCondition[] = "page_namespace != '" . NS_WIKIA_FORUM_BOARD_THREAD . "'";
+	}
+
+	/**
+	 * IRIS-5184: Exclude outgoing links in Forum content from Special:WantedPages report
+	 * @see WantedPagesPage::getQueryInfo()
+	 *
+	 * @param IContextSource $context
+	 * @param array $queryInfo
+	 */
+	public static function onWantedPagesGetQueryInfo( IContextSource $context, array &$queryInfo ) {
+		$queryInfo['conds'][] = "pg2.page_namespace != '" . NS_WIKIA_FORUM_BOARD_THREAD . "'";
+	}
+
+	/**
+	 * @param array $vars JS variables to be added at the bottom of the page
+	 * @param $scripts
+	 */
+	public static function addDiscussionJsVariable( array &$vars, &$scripts ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$scripts". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

[project-id]: wikia:app